frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -352,6 +377,10 @@ const DeviceUserPage = ({
     const isSoftwareEnabled = !!globalConfig?.features
       ?.enable_software_inventory;
 
+    const isDarwinHost = host?.platform === "darwin";
+    const isIosOrIpadosHost =
+      host?.platform === "ios" || host?.platform === "ipados";
```

Comment:
Do we have utility functions someplace for this already?

Reply:
just looked and i dont see one. I will add this the next time i run into this check.

Reply:
I happened across one working on the other story: `isIPadOrIPhone` in `platform.ts`

Reply:
ah thanks will make the change